build-system/tasks/e2e/puppeteer-controller.js

```diff
@@ -559,7 +559,8 @@ class PuppeteerController {
    * @override
    */
   async getActiveElement() {
-    const getter = () => document.activeElement;
+    const root = await this.getRoot_();
+    const getter = () => root.ownerDocument.activeElement;
```

Comment:
I think you you need to pass root to `this.evaluate` and as a parameter in the `getter`, since the `getter` runs in the context of the page being tested

```js
const root = await this.getRoot_();
const getter = root => root.ownerDocument.activeElement;
const element = await this.evaluate(getter, root);

Reply:
this was sloppy copy paste, Done.